bands/site_sections/band_admin.py

```diff
@@ -24,6 +24,21 @@ def mark_as_band(self, session, group_id):
             'message': '{} has been marked as a band'.format(group.name)
         }
 
+    @ajax
+    def remove_as_band(self, session, group_id):
+        group = session.group(group_id)
+        if not group.leader:
+            return {'message': '{} does not have an assigned group leader'.format(group.name)}
```

Comment:
Just to clarify, we DO require a leader in order to mark the group as being a band, on the logic that the music department wants a primary contact and we want a particular person to be getting the notification emails about the band checklist.

We just shouldn't require them to still have a leader if we're un-setting their status as being a band.